simpledb/flush.go

```diff
@@ -50,6 +51,7 @@ func executeFlush(db *DB, flushAction memStoreFlushAction) error {
 	err = memStoreToFlush.FlushWithTombstones(
 		sstables.WriteBasePath(writePath),
 		sstables.WithKeyComparator(db.cmp),
+		sstables.WriteBufferSizeBytes(16*1024),
```

Comment:
Think that it's enough no ? Should make some benchmark. But 4Mb for each flush is for my case a huge waste.

Reply:
feel free to make it configurable as well, but 16k is absolutely fine